
[31;01mException occurred:[39;49;00m
  File "/home/luc/hgwork/lino/lino/utils/choosers.py", line 230, in uses_simple_values
    choices = list(fld.choices)
AttributeError: 'DummyField' object has no attribute 'choices'
[31;01mThe full traceback has been saved in /tmp/sphinx-err-DwlNtc.log, if you want to report the issue to the developers.[39;49;00m
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-users/>,
or report them in the tracker at <http://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
